packages/nameguard-react/src/components/LoadingReportIcon/LoadingReportIcon.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { Tooltip } from "../Tooltip/Tooltip";
+import { ENSName } from "@namehash/ens-utils";
+import { RatingIcon, RatingIconSize } from "../Report/RatingIcon";
+import { RatingLoadingIcon } from "../icons/RatingLoadingIcon";
+
+type LoadingReportIconProps = {
```

Comment:
Please see my comment below about naming. Whatever we decide there, let's align the name of this type to match.